web/src/components/njwds/Hero.tsx

```diff
@@ -81,7 +81,7 @@ export const Hero = (props: Props): ReactElement => {
                   isDesktopAndUp ? "width-80 margin-top-15" : "maxw-tablet margin-auto margin-top-205"
                 } padding-x-4 tablet:padding-x-10 desktop:padding-x-0`}
               >
-                <img src="/img/Landing-Hero-people.png" alt="People" />
+                <img src="/img/Landing-Hero-people.png" alt="" />
```

Comment:
Is there a reason this didn't get the role presentation as well?

Reply:
These were the guidelines that I followed:

- If image was both decorative and illustrative in nature, it got both alt="" and role="presentation".
- If image was decorative, it got alt="".
- If image was informative, its alternative text was updated.

That was my read from the W3C page on this, as the only example that was given for role="presentation" was illustrative in nature; the rest did not mention that. I'm open to other interpretations though: if you think all should get role="presentation", I'll defer to your expertise!